cypress/e2e/chi-js/sidenav.cy.js

```diff
@@ -1,145 +1,496 @@
+const ACTIVE_CLASS = '-active';
+const CLOSE_CLASS = '-close';
+const EXPANDED_CLASS = '-expanded';
+const ACCORDION_CLASSES = {
+  ITEM: 'chi-accordion__item',
+  CONTENT: 'chi-accordion__content'
+};
+const SIDENAV_LIST_CLASS = 'chi-sidenav__list';
 const drawerLiTitleSelector =
   ' .chi-sidenav__drawer-list li div.chi-sidenav__title';
 const drawerDropDownTitle = '.chi-sidenav__drawer-item-list ul.chi-tabs li a';
 const drawerLiMainMenuTitleSelector =
   ' .chi-sidenav__drawer-list li span.chi-sidenav__title';
 
-describe('Sidenav', function() {
-  before(() => {
-    cy.visit('tests/js/sidenav.html');
+const hasClassAssertion = (el, value) => {
+  cy.get(el).should('have.class', value);
+};
+
+describe('Legacy Sidenav', () => {
+  before (() => {
+    cy.visit ('tests/js/sidenav.html');
   });
 
-  beforeEach(() => {
-    cy.get('[data-cy="sidenav-legacy"]').as('legacySidenavSelector');
-  });
-
-  describe('check that click on 1st level opens 2nd-level menu drawer', () => {
-    [/^Active Item$/, /^Menu Item$/].forEach(menuText => {
-      it(`check "${menuText}" opens 2nd-level menu drawer`, () => {
-        cy.get('@legacySidenavSelector').within(() => {
-          cy.get('.chi-sidenav__list li')
-            .contains(menuText)
-            .as('firstLevelElement');
-          cy.get('@firstLevelElement')
-            .should('have.attr', 'href')
-            .then(href => {
-              cy.get('' + href).as('drawer');
-              cy.get('@firstLevelElement')
-                .click()
-                .wait(550);
-              cy.get(`${href}.-active`).should('be.visible');
+  beforeEach (() => {
```

Comment:
Remove spaces in all file after function name:

```suggestion
  beforeEach(() => {
```